### DISCUSSION TITLE
Matterbridge : Nextcloud Talk < > Email support?

### DISCUSSION THREAD

**Discussion by benjamindahl** [42wim/matterbridge#1583] [Help / Q&A] [answered]
Hello, is there any way or are there any plans to bridge a Nextcloud Talk to E-Mail? For would need such functionality for different work groups.

Cheers
Benny

**Comment by qaisjp** [comment, marked as answer]
There are no such plans at the moment. Additionally, email isn't a chat platform so I don't think this would ever be supported first-class. You could implement this via the [API](https://github.com/42wim/matterbridge/wiki/API), though.

**Reply by benjamindahl** [reply]
ok. Since I'm not a developer I would need to find somebody who would do it.
